handlers.go

```diff
@@ -47,31 +49,41 @@ func setSessionCookie(w http.ResponseWriter, id int) {
 }
 
 func signupSubmit(w http.ResponseWriter, r *http.Request) {
+	username := r.FormValue("username")
+	// TODO(@seoyoungcho213): Validate user data way better here.
+	if username == "" {
+		log.Println("Error adding user {" + username + "} to database")
+		if _, err := io.WriteString(w, "username can't be blank"); err != nil {
+			log.Println("Error writing signup username format error -", err)
+		}
+		return
+	}
+	email := r.FormValue("email")
+	// TODO(@seoyoungcho213): read about mail.ParseAddress' return type and use it to add first+last name formatting too
+	if _, err := mail.ParseAddress("<" + email + ">"); err != nil {
```

Comment:
i'm trying to handle validation using HTMX Form Validation since we have to check for every input.

check https://fandiilham.medium.com/go-htmx-form-validation-f4c40bc46e95

Reply:
you can then rebase onto this pr and add your backend actual validation in handlers.go, like for example we need to also validate the birth date
this pr doesnt complete validation it only enables ui and serves as an example of how we are going to pass errors to the end user's ui